pkg/component/component.go

```diff
@@ -44,14 +46,15 @@ func CreateFromGit(client *occlient.Client, name string, ctype string, url strin
 }
 
 // CreateFromDir create new component with source from local directory
-func CreateFromDir(client *occlient.Client, name string, ctype string, dir string, applicationName string) error {
+func CreateFromDir(client *occlient.Client, name string, ctype string, dir string, applicationName string, sourceType string) error {
```

Comment:
It might be good to mention in the comment that `sourceType` can be either `local` or `binary` and nothing else.